extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -92,9 +92,14 @@ class AmpSocialShare extends AMP.BaseElement {
       // mailto:, whatsapp: protocols breaks when opened in _blank on iOS Safari
       const isMailTo = /^mailto:$/.test(parseUrl(href).protocol);
       const isWhatsApp = /^whatsapp:$/.test(parseUrl(href).protocol);
+      const isSMS = /^sms:$/.test(parseUrl(href).protocol);
       const isIosSafari = this.platform_.isIos() && this.platform_.isSafari();
-      this.target_ = (isIosSafari && (isMailTo || isWhatsApp))
+      this.target_ = (isIosSafari && (isMailTo || isWhatsApp || isSMS))
           ? '_top' : '_blank';
+      if (isSMS && isIosSafari) {
+        // http://stackoverflow.com/a/19126326
+        this.href_ = this.href_.replace('?body', '&body');
```

Comment:
Looks like we can use` sms://${NUMBER}?&body=${BODY}` (worked on iOS 10 and whatever android phone I have)